app.py

```diff
@@ -108,6 +108,16 @@ async def submit(request: Request) -> RedirectResponse:
 
 
 if __name__ == "__main__":
+    if not OUTPUT_DIR.exists():
+        print(f"Creating output directory at {OUTPUT_DIR}")
+        OUTPUT_DIR.mkdir(parents=True, exist_ok=True)
```

Comment:
Probably worth removing `exist_ok` here, as you're already checking whether it exists above.
```suggestion
        OUTPUT_DIR.mkdir(parents=True)
```

Reply:
We are checking above, however removing `exist_ok` introduces a race condition if something creates the directory in the meanwhile.